chart/kubeapps/values.yaml

```diff
@@ -1475,6 +1475,8 @@ allowNamespaceDiscovery: true
 ##   certificateAuthorityData: LS0tLS1CRUdJ...
 ##   serviceToken: ...
 ##   isKubeappsCluster: true
+##   pinnipedConfig: true
```

Comment:
But it's not a boolean value, rather an object?
```suggestion
##   pinnipedConfig:
##     enable: true
```

Reply:
Oops, my bad, I did it quickly and overlapped `pinnipedConfig` and `enable: true` in my mind. Thanks for fixing it!